pkgs/applications/blockchains/bisq-desktop/default.nix

```diff
@@ -60,16 +40,26 @@ stdenv.mkDerivation rec {
       name = "Bisq";
       exec = "bisq-desktop";
       icon = "bisq";
-      desktopName = "Bisq";
+      desktopName = "Bisq ${version}";
       genericName = "Decentralized bitcoin exchange";
-      categories = "Network;Utility;";
+      categories = "Network;P2P;";
     })
   ];
 
   unpackPhase = ''
     dpkg -x $src .
   '';
 
+  buildPhase = ''
+    # Replace the embedded Tor binary (which is in a Tar achive)
+    # with one from Nixpkgs.
+
+    mkdir -p native/linux/x64/
+    cp ${tor}/bin/tor ./
```

Comment:
It's better not to rely on the property that `tor` still works when copied to a new location. A more robust solution is to package a tiny script that `exec`s the original nixpkgs tor binary in its normal location.